### PR TITLE
Add Resume blog post first version

### PR DESCRIPTION
**Pull request by habanhero**
Wasn't able to review it through this link:
https://habanhero.github.io/source.coveo.com/

But spelled checked and used only tried and true markup.

### CODE REVIEW

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  - Resume
+
+author:
+  name: Nicolas Pelletier
+  bio: Director R&D - Coveo Platform and Geek Searcher
+  twitter: habanhero
+  image: npelletier.jpg
+---
+
+We have many interns right now at Coveo. For the summer, this process starts in February when we get over one hundred applications through multiple universities. All those applications had one thing in common, **The Resume**.
```
Comment (malaporte) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: Resumes? Unsure.

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+Best recommendation is to keep your resume as short as possible. Intern or fresh out of school? 1 page, top 2 is you have had many internships, projects, open source contributions, etc.
+
+## Keep it electronic!
+
+Always remember when you apply for what job you are applying. Want to come and code with us? Awesome. The only resume you have is paper? Awesome. As in I just missed my plane awesome.
+
+The electronic version I can share it automatically with everybody I need to, have it on my laptop if I ever need to, search for it, search in it.
+To achieve the same result with the paper version, I need to scan it, OCR it, then do all the other steps. This is too much of a time investment in turning a document into bits and bytes for somebody that should help us solve gigantic programming challenges.
+
+Oh, and the electronic version cannot get covered in spilled wine. Just saying.
```
Comment (malaporte) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: Better to drown the laptop in wine, eh?

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+## Keep your metadata clean!
+
+You know those special properties in Words and PDF; title author etc.? Be sure to fill them in. The more the document goes into software, the more chances this meta information ends up being used. 
+
+You made your resume based on your mothers because you liked her template? If you don’t fill the properties, you might not only end up using her fonts. Oh, you liked her recipes template. Even better.
```
Comment (malaporte) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: mother's ?

Comment (malaporte) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: Oh and I've seen worst than a recipe name. @marcsanfacon probably remembers.

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+Over the years, I’ve read hundreds of them and have therefore accumulated a good list of what you should and shouldn’t do. 
+
+<!-- more -->
+
+Keep in mind that this will be aimed more toward people looking for internships, but most details also apply to the general public. Also, this is based on personal experience; I've not consulted a long list of experts. Finally, I have a long list of notes. This is only a subset of quick things around submitting your resume.
+
+
+## Keep it short!
+
+[A study in 2012](http://www.forbes.com/sites/susanadams/2012/03/26/what-your-resume-is-up-against/) has shown that the average recruiter will spend 6.25 seconds (yes, seconds) before deciding if further investigation is required, or if this resume goes into the (virtual) recycling bin. This is probably less time than you invested in your latest tweet.
```
Comment (pastjean) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: You talk about the study but link the article talking about the study on forbes. (Sorry graduate student habits)

File: _posts/2016-06-22-of-reading-too-many-resumes.md (hunk 2 above)
Comment (wfortin) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: Hahaha this one is funny! (cause it's a true story 😛 )

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+Best recommendation is to keep your resume as short as possible. Intern or fresh out of school? 1 page, top 2 is you have had many internships, projects, open source contributions, etc.
+
+## Keep it electronic!
+
+Always remember when you apply for what job you are applying. Want to come and code with us? Awesome. The only resume you have is paper? Awesome. As in I just missed my plane awesome.
```
Comment (wfortin) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: As in _I just missed my plane_ awesome.

Might want to emphasize the I juste missed my plane in italic or something.

**Comment by wfortin** [comment]
Great article, i really like the tone of it!

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+<!-- more -->
+
+Keep in mind that this will be aimed more toward people looking for internships, but most details also apply to the general public. Also, this is based on personal experience; I've not consulted a long list of experts. Finally, I have a long list of notes. This is only a subset of quick things around submitting your resume.
+
+
+## Keep it short!
+
+[A study in 2012](http://www.forbes.com/sites/susanadams/2012/03/26/what-your-resume-is-up-against/) has shown that the average recruiter will spend 6.25 seconds (yes, seconds) before deciding if further investigation is required, or if this resume goes into the (virtual) recycling bin. This is probably less time than you invested in your latest tweet.
+
+Best recommendation is to keep your resume as short as possible. Intern or fresh out of school? 1 page, top 2 is you have had many internships, projects, open source contributions, etc.
```
Comment (erocheleau) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: > 1 page, top 2 is you have had many internships, projects, open source contributions, etc.

-is- **if** you have had ?

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+You made your resume based on your mothers because you liked her template? If you don’t fill the properties, you might not only end up using her fonts. Oh, you liked her recipes template. Even better.
+
+## Keep your filename clean!
+
+There are a few subtopics here. 
+
+First, welcome to 1995. Long gone the 8 char limit. You can use more than a word to name your document. Actually, please do.
+
+Second, choose a name in a way that scales for the person receiving it. You may decide to name the resume with the company you are sending it to (so you know which version you sent). But think this from my shoes. I get 100 resume all named “Coveo.pdf”. I then need to do some renaming to avoid them clashing if I drop them in the same folder. *2016 Summer - John Doe for Coveo.pdf* Now we are talking. I can guess from this you'd be able to write proper email title, variable and function name and code a distributed resource locking mechanism.
```
Comment (erocheleau) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: But think this from my shoes.

> But think about this from my point of view.

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+Last, don’t PascalCase your words. I may search by name. I have a better chance to find it if the OS or the system can recognize the words. Separate them with a space, hyphen or underscore (this last one does not always work well in all systems).
+
+## The Cover Letter
+
+There are two philosophies here. Those who don’t look at the cover letter (some study say it’s 90% of the market), and those who find it’s an important way to know a candidate. I think they are a good way to know the motivations of a candidate (or lack thereof).
+
+You have a few options, in my preferred order:
+- A personalized cover letter where you clearly took the time to write something unique for us
```
Comment (erocheleau) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: Je pense que la list ne render pas bien ici

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+  image: npelletier.jpg
+---
+
+We have many interns right now at Coveo. For the summer, this process starts in February when we get over one hundred applications through multiple universities. All those applications had one thing in common, **The Resume**.
+
+Over the years, I’ve read hundreds of them and have therefore accumulated a good list of what you should and shouldn’t do. 
+
+<!-- more -->
+
+Keep in mind that this will be aimed more toward people looking for internships, but most details also apply to the general public. Also, this is based on personal experience; I've not consulted a long list of experts. Finally, I have a long list of notes. This is only a subset of quick things around submitting your resume.
```
Comment (lbenie) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: Finally... Unsure

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+
+[A study in 2012](http://www.forbes.com/sites/susanadams/2012/03/26/what-your-resume-is-up-against/) has shown that the average recruiter will spend 6.25 seconds (yes, seconds) before deciding if further investigation is required, or if this resume goes into the (virtual) recycling bin. This is probably less time than you invested in your latest tweet.
+
+Best recommendation is to keep your resume as short as possible. Intern or fresh out of school? 1 page, top 2 is you have had many internships, projects, open source contributions, etc.
+
+## Keep it electronic!
+
+Always remember when you apply for what job you are applying. Want to come and code with us? Awesome. The only resume you have is paper? Awesome. As in I just missed my plane awesome.
+
+The electronic version I can share it automatically with everybody I need to, have it on my laptop if I ever need to, search for it, search in it.
```
Comment (lbenie) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: With the electronic version ?

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+## Keep your filename clean!
+
+There are a few subtopics here. 
+
+First, welcome to 1995. Long gone the 8 char limit. You can use more than a word to name your document. Actually, please do.
+
+Second, choose a name in a way that scales for the person receiving it. You may decide to name the resume with the company you are sending it to (so you know which version you sent). But think this from my shoes. I get 100 resume all named “Coveo.pdf”. I then need to do some renaming to avoid them clashing if I drop them in the same folder. *2016 Summer - John Doe for Coveo.pdf* Now we are talking. I can guess from this you'd be able to write proper email title, variable and function name and code a distributed resource locking mechanism.
+
+Last, don’t PascalCase your words. I may search by name. I have a better chance to find it if the OS or the system can recognize the words. Separate them with a space, hyphen or underscore (this last one does not always work well in all systems).
```
Comment (lbenie) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: Last
> Lastly

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+
+You have a few options, in my preferred order:
+- A personalized cover letter where you clearly took the time to write something unique for us
+- No cover letter (Close second option)
+- A cover text sang by a dancing clown while trowing pies and confetti running around the office
+- A generic cover letter that you could have sent elsewhere if you'd replace 3-4 words
+
+So if you don't have time to make a serious cover letter, then avoid going for the quick fix.
+
+I'll add that a cover letter is an introduction to your resume. When sending it by email, you don't need a PDF cover letter. Your email IS the introduction to your resume. It is your cover letter. Don't introduce your introduction to your resume, I'll feel over-introduced.
```
Comment (lbenie) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: More emphasis on `IS` ?

> **IS**

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+
+## So!
+
+Here you go. A few quick things around submitting your resume. No rocket science, but details that can make a difference when we are going through piles of resumes and there is not much separating candidate #19 from #20 in line.
```
Comment (lbenie) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: A few quick things around submitting your resume.
> A few quick tips to keep in mind before submitting resume.

Comment (lbenie) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: but details that
> but details can

File: _posts/2016-06-22-of-reading-too-many-resumes.md (hunk 11 above)
Comment (lbenie) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: this last one
> ~~this~~ the last one

File: _posts/2016-06-22-of-reading-too-many-resumes.md (hunk 1 above)
Comment (habanhero) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: https://en.wiktionary.org/wiki/resumes

File: _posts/2016-06-22-of-reading-too-many-resumes.md (hunk 3 above)
Comment (habanhero) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: DM me details plz :D

File: _posts/2016-06-22-of-reading-too-many-resumes.md [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+Last, don’t PascalCase your words. I may search by name. I have a better chance to find it if the OS or the system can recognize the words. Separate them with a space, hyphen or underscore (this last one does not always work well in all systems).
+
+## The Cover Letter
+
+There are two philosophies here. Those who don’t look at the cover letter (some study say it’s 90% of the market), and those who find it’s an important way to know a candidate. I think they are a good way to know the motivations of a candidate (or lack thereof).
+
+You have a few options, in my preferred order:
+- A personalized cover letter where you clearly took the time to write something unique for us
+- No cover letter (Close second option)
+- A cover text sang by a dancing clown while trowing pies and confetti running around the office
```
Comment (GuiSim) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: throwing, with an `h`.

**Comment by GuiSim** [comment]
LGTM ! :shipit:

File: _posts/2016-06-22-of-reading-too-many-resumes.md (hunk 4 above)
Comment (habanhero) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: The link to the reseach is dead. I'm open to suggestions.

File: _posts/2016-06-22-of-reading-too-many-resumes.md (hunk 11 above)
Comment (habanhero) [inline _posts/2016-06-22-of-reading-too-many-resumes.md]: Lastly, TIL

**Pull request merged**